### PR TITLE
feat(components/molecule/progressSteps): add content type

### PR DESCRIPTION
**Pull request by beatrizip**
### Types of changes
Added new prop for tab content adjustment

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Only
- [ ] Refactor

### Description, Motivation and Context
When we have a step form, and one step has more content than the others, we need to adjust the tab content

### Screenshots - Animations
THEN:
Captura de pantalla 2022-03-07 a las 11 46 46

NOW
Captura de pantalla 2022-03-07 a las 11 47 08

### CODE REVIEW

File: components/molecule/progressSteps/src/index.js [hunk 1]
```diff
@@ -86,7 +88,7 @@ const MoleculeProgressSteps = ({
       return (
         <div
           key={index}
-          className={cx(`${CLASS_CONTENT}-item`, {
+          className={cx(`${CLASS_CONTENT}-item `, {
```
Comment (andresin87) [inline components/molecule/progressSteps/src/index.js]: i this this extra space is no needed

File: components/molecule/progressSteps/src/index.js [hunk 2]
```diff
@@ -113,7 +115,9 @@ const MoleculeProgressSteps = ({
       >
         {extendedChildren}
       </div>
-      <div className={CLASS_CONTENT}>{childrenContent}</div>
+      <div className={`${CLASS_CONTENT} ${contentStyle}`}>
```
Comment (andresin87) [inline components/molecule/progressSteps/src/index.js]: cx(CLASS_CONTENT, contentStyle)

File: components/molecule/progressSteps/src/index.js [hunk 3]
```diff
@@ -144,5 +151,6 @@ export {
   MoleculeProgressStep,
   STATUSES,
   STATUSES as moleculeProgressStepsStatuses,
-  PROGRESS_BAR_JUSTIFY_CONTENT as moleculeProgressStepsJustifyContentBar
+  PROGRESS_BAR_JUSTIFY_CONTENT as moleculeProgressStepsJustifyContentBar,
+  CONTENT_STYLE as MoleculeProgressContentStyle
```
Comment (andresin87) [inline components/molecule/progressSteps/src/index.js]: Mayusc for clases or components.
change it for `moleculeProgressContentStyle`

File: components/molecule/progressSteps/src/styles/index.scss [hunk 4]
```diff
@@ -1,4 +1,5 @@
 $base-class: '.sui-MoleculeProgressSteps';
+$content-class: $base-class + '-content';
```
Comment (andresin87) [inline components/molecule/progressSteps/src/styles/index.scss]: use same patters than many others:
`$content-class: '#{base-class}-content';`

File: components/molecule/progressSteps/test/index.test.js [hunk 5]
```diff
@@ -37,6 +38,7 @@ describe(json.name, () => {
       STATUSES,
       moleculeProgressStepsStatuses,
       moleculeProgressStepsJustifyContentBar,
+      MoleculeProgressContentStyle,
```
Comment (andresin87) [inline components/molecule/progressSteps/test/index.test.js]: minusc

File: components/molecule/progressSteps/test/index.test.js [hunk 6]
```diff
@@ -28,6 +28,7 @@ describe(json.name, () => {
       'STATUSES',
       'moleculeProgressStepsStatuses',
       'moleculeProgressStepsJustifyContentBar',
+      'MoleculeProgressContentStyle',
```
Comment (andresin87) [inline components/molecule/progressSteps/test/index.test.js]: minusc

**Review by andresin87 — APPROVED** [review]

**Review by jcger — APPROVED** [review]

**Review by manjarcode — APPROVED** [review]

**Pull request merged**